Essentials/src/com/earth2me/essentials/ISettings.java

```diff
@@ -310,4 +310,6 @@
     boolean isCompassTowardsHomePerm();
 
     boolean isAllowWorldInBroadcastworld();
+
+	boolean allowSellNamedItems();
```

Comment:
Minor nag but the indentation here is inconsistent.

Reply:
I guess you mean the tab instead of spaces? I didn't even notice :/